libcodechecker/libhandlers/analyze.py

```diff
@@ -129,6 +129,7 @@ def is_ctu_capable():
     parser.add_argument('-o', '--output',
                         dest="output_path",
                         required=True,
+                        default=argparse.SUPPRESS,
```

Comment:
Why did we suppress this?

Reply:
Because this removes the `(default: None)` message from the help page.